react-hot-button/my-app/src/ButtonContainer.tsx

```diff
@@ -0,0 +1,12 @@
+type Props = {
+  Class: string | undefined;
```

Comment:
It would be best to name it `className` so that it matches the name you would use in the HTML elements. That makes the code cleaner and easier to understand. Also, since your prop names should start with lowercase, it means you don't have to use `class` as a prop name!

Also, I like that you are allowing Class to be undefined. However, that's not as convenient as it may seem. You should instead make it optional, with `?`. Here's why:

With `type Props = { className: string | undefined };`, when I render your component, I am _required_ to include a prop named `className`, even if I don't want to provide one. I would have to do this like:
`<ButtonContainer className={undefined} />`.

If you instead make it optional, with `type Props = { className?: string }` then I do not have to include the `className` prop if I don't need it. It would then look like:
`<ButtonContainer />`.